main.py

```diff
@@ -0,0 +1,31 @@
+from fastapi import FastAPI

```

Comment:
please read [this part](https://peps.python.org/pep-0008/#imports) of PEP 8  and change your imports